server/sonar-web/src/main/js/apps/issues/facets/assignee-facet.js

```diff
@@ -111,7 +111,7 @@ export default CustomValuesFacet.extend({
   },
 
   sortValues (values) {
-    return _.sortBy(values, function (v) {
+    return _.sortBy(values, v => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)